package.json

```
@@ -34,6 +34,7 @@
         ],
         "exclude": [
             "src/gen/*/**.ts",
+            "src/gen/**.ts",
```

I think this makes the line above it unnecessary.

---

If I remove that line it prints coverage for all of the generated API files. I agree that semantically I feel like you are correct, but in practice it doesn't work right.

---

Oh wait. I think it's because the existing pattern should be `src/gen/**/*.ts`. That fixes it for me locally.

---

As far as I understand globs, the `**.ts` would not be different than `*.s` and `**/*.ts` would pick up all ts files including subdirs. I've actually tested it right now with ls. So this should be enough:

```suggestion
        "exclude": [
            "src/gen/**/*.ts",
```

---

done.